flutter_local_notifications/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [19.0.0-dev.4]
+
+* [Windows] Reworked the APIs around custom images and audio. Use the new `.asset()` or `.network()` constructors. This is a breaking change from `19.0.0-dev.1`. 
```

Comment:
Looks like there's breaking changes between different prereleases versions. Can you update the changelog entry to be more explicit in calling these out?

Reply:
Oh, sorry, did this in the main package and not the sub-package. Fixed!

Reply:
Oh wait did you want me to elaborate more on the top-level package as well?

Reply:
It should apply to both. I was referring to a couple of things though on this though

- breaking changes are indicated via `**Breaking changes**` for multiple things or `**Breaking change**` when there's a single breaking change. There are existing examples in other changelog entries
- details on what's been removed
- a note that the example app being updated to demonstrate usages could be worth mentioning too

Reply:
Ah okay. I figured since it was just a breakage on a _pre-release_ it was okay, but I added that stuff now.